src/presence_analyzer/tests.py

```diff
@@ -38,15 +38,21 @@ def tearDown(self):
         """
         Get rid of unused objects after each test.
         """
-        pass
 
     def test_mainpage(self):
         """
         Test main page redirect.
         """
         resp = self.client.get('/')
         self.assertEqual(resp.status_code, 302)
-        assert resp.headers['Location'].endswith('/presence_weekday.html')
+        assert resp.headers['Location'].endswith('/presence_weekday')
+
+    def test_template_router(self):
+        """
+        Test template_router redirect.
+        """
+        resp = self.client.get('/presence_weekday')
+        self.assertEqual(resp.status_code, 200)
```

Comment:
Just for peace of mind, this could check content of the response for something that is in the template